src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public abstract class Task {
+    protected String desc;
+    protected boolean isDone;
+    protected TaskType taskType;
+
+    public Task(String desc) {
+        this.desc = desc;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Slight styling issue. The comment should probably be in the javadoc instead of inline. Writing it in javadoc would help you generate beautiful documentation too 😄